lucene/BUILD.md

```diff
@@ -2,78 +2,72 @@
 
 ## Basic steps:
   
-  0. Install OpenJDK 11 (or greater), Ant 1.8.2+, Ivy 2.2.0
-  1. Download Lucene from Apache and unpack it
-  2. Connect to the top-level of your Lucene installation
+  0. Install OpenJDK 11 (or greater), Gradle 6.4.1
```

Comment:
We have gradle wrapper in the repo, installing Gradle itself is still required?

Reply:
No, it isn't. Gradle wrapper will download itself. We should *not* mention any external installation requirements because different branches will be on different gradle version (and may not be interoperable). The wrapper handles this transparently.

Reply:
@ErickErickson 
I opened a PR on your repository to refine lucene/BUILD.md: https://github.com/ErickErickson/lucene-solr/pull/1
Could you check it?